metadata/0x53b2d22d07E069a3b132BfeaaD275b10273d381E.json

```diff
@@ -42,7 +42,7 @@
       "Issuer": "NS Pool LLC"
     },
     "discourseLink": "https://discourse.centrifuge.io/t/community-introduction-new-silver/401",
-    "asset": "Real estate bridge loans",
+    "asset": "Real Estate Bridge Loans",
```

Comment:
Changed in discussion with bizdev to better align collateral types of all pools.